### PR TITLE
Load a tuned OssSecurityScore from a file

### LINKED ISSUE #36: Load OssSecurityScore from a file
Test vectors for scores have been recently introduced. Now we can tune scores separately from ratings. That gives more flexibility.

The `RatingRepository` loads an instance of `OssSecurityRating` from a file including `OssSecurityScore`. The `OssScores` also provides an instance of `OssSecurityRating` which has all weights set to 1.0. That's not correct. Instead, the `RatingRepository` should load scores from files. Then, the `RatingRepository` should offer the loaded scores to the client code.

Here is what needs to be done:

- Update the `RatingRepository` class to load scores.
- Store `OssSecurityScore` to a file.
- Remove `OssScores` class.

### PR DESCRIPTION
**Pull request by artem-smotrakov**
Here is a list of main updates:

- Stored a tuned instance of `OssSecurityScore` to a file.
- Updated `RatingRepository` to load a tuned instance of `OssSecurityScore` from a file.
- Updated `OssSecurityScoreTestVectors.ipynb` to tune the open-source security score
- Renamed `get()` to `rating()` in `RatingRepository`.
- Removed redundant test vectors for `OssSecurityRating`.
- Removed the `OssScore` class.
- Updated tests.

This closes #36

### CODE REVIEW

File: src/main/java/com/sap/sgs/phosphor/fosstars/model/RatingRepository.java [hunk 1]
```diff
@@ -20,6 +23,10 @@
  */
 public class RatingRepository {
 
+  private interface RatingFactory {
+    Rating create() throws IOException;
+  }
+
   /**
    * A logger.
    */
```
Comment (sourabhsparkala) [inline src/main/java/com/sap/sgs/phosphor/fosstars/model/RatingRepository.java]: Can we replace MAPPER.enableDefaultTyping() (Depracated) to  MAPPER.activateDefaultTyping(MAPPER.getPolymorphicTypeValidator());?

Comment (artem-smotrakov) [reply src/main/java/com/sap/sgs/phosphor/fosstars/model/RatingRepository.java]: Yeah, that's a good point. I agree. Yesterday I created #37 to address that. I would prefer to fix it separately.

Comment (artem-smotrakov) [reply src/main/java/com/sap/sgs/phosphor/fosstars/model/RatingRepository.java]: I am not sure it `MAPPER.activateDefaultTyping(MAPPER.getPolymorphicTypeValidator());` is enough. Maybe we'll need to configure the validator more carefully.

Comment (sourabhsparkala) [reply src/main/java/com/sap/sgs/phosphor/fosstars/model/RatingRepository.java]: 👍

**Pull request merged**